### DISCUSSION TITLE
[Bug]: Open AI can not work

### DISCUSSION THREAD

**Discussion by jiantao88** [danny-avila/LibreChat#499] [Troubleshooting]
### Contact Details

jiantao8866@gmail.com

### What happened?

I added the openai key to run in docker, but the chartGPT question returns an error

### Steps to Reproduce

1. input open ai key to .env
2. docker-compose build 
3. docker-compose up

### What browsers are you seeing the problem on?

Chrome

### Screenshots

<img width="1041" alt="image" src="https://github.com/danny-avila/LibreChat/assets/7411102/faba7120-ed95-4560-9e49-e59963b13195">

<img width="1000" alt="image" src="https://github.com/danny-avila/LibreChat/assets/7411102/0aecbc65-68c6-4b90-8d28-93e2d9c33e9b">

<img width="870" alt="image" src="https://github.com/danny-avila/LibreChat/assets/7411102/ba2561a6-1c79-4005-9d3e-41f8022f4098">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
can you provide your .env file while removing any sensitive information, which is found in the root of the project directory?

also, if you have docker desktop, it would useful to see what env variables its using, again removing sensitive info

![image](https://github.com/danny-avila/LibreChat/assets/110412045/14aad6d9-2b06-4a65-baff-1251dadf30a4)

Click on LibreChat

Then click on Inspect

![image](https://github.com/danny-avila/LibreChat/assets/110412045/af58bc97-ac2e-4e70-853c-e5d05a0ed0fb)

**Comment by danny-avila** [comment]
note that OPENAI_KEY changed to OPENAI_API_KEY to conform with standards

**Comment by jiantao88** [comment]
[env.txt](https://github.com/danny-avila/LibreChat/files/11715145/env.txt)

**Comment by jiantao88** [comment]
<img width="567" alt="image" src="https://github.com/danny-avila/LibreChat/assets/7411102/be4f9c69-fd72-4fe0-87f8-852463584e7f">

**Comment by danny-avila** [comment]
You may need a proxy to access OpenAI 

https://github.com/waylaidwanderer/node-chatgpt-api/issues/144
https://github.com/waylaidwanderer/node-chatgpt-api/issues/177

Are you able to access OpenAI servers outside of this app?
